sklearn/ensemble/tests/test_forest.py

```diff
@@ -1791,3 +1791,65 @@ def test_round_samples_to_one_when_samples_too_low(class_weight):
         n_estimators=10, max_samples=1e-4, class_weight=class_weight, random_state=0
     )
     forest.fit(X, y)
+
+
+@pytest.mark.parametrize(
+    "make_data, Forest",
+    [
+        (datasets.make_regression, RandomForestRegressor),
+        (datasets.make_classification, RandomForestClassifier),
+    ],
+)
+def test_missing_values_is_resilience(make_data, Forest):
```

Comment:
Should it be? From an english point of view `.._is_resilience` sounds weird, and/or I can't work out what it means :)

```suggestion
def test_missing_values_is_resilient(make_data, Forest):
```